src/EA.Weee.Web/Areas/Admin/ViewModels/Producers/ReturnedViewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace EA.Weee.Web.Areas.Admin.ViewModels.Producers
+{
+    public class ReturnedViewModel
```

Comment:
@will-hope98 can we re-use the ProducerDetailsScheme model?

Reply:
@JohnPadgettCivica As discussed leaving as it is